DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi/FrontEnd/src/client/app/stats/stats.factory.js

```diff
@@ -0,0 +1,40 @@
+﻿(function () {
+    'use strict';
+    var module = angular.module('app.stats');
```

Comment:
Only C# code. We have separation between FE and BE please don't mix in one pull request this two concerns.